src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -402,6 +402,14 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>La valeur du masque de réseau doit être comprise entre {{ min }} et {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Le nom du fichier est trop long. Il doit contenir au maximum {{ filename_max_length }} caractère.|Le nom de fichier est trop long. Il doit contenir {{ filename_max_length }} caractères ou moins.</target>
```

Comment:
Yes, that's beacause you can't have less than 1 character in a filename!

Reply:
Well one can write `au maximum 1 caractère`, `au maximum 2 caractères`, `1 caractère ou moins` or `2 caractères ou moins`; these are synonyms.

Wouldn’t it be simpler to use the same wording for singular and plural?

FWIW I would use `au maximum` because it’s easy to confuse `1 caractère **ou** moins` and `1 caractère **au** moins`, which are opposite:

```suggestion
                <target>Le nom du fichier est trop long. Il doit contenir au maximum {{ filename_max_length }} caractère.|Le nom de fichier est trop long. Il doit contenir au maximum {{ filename_max_length }} caractères.</target>
```

WDYT?

Reply:
I'm ok with this, thanks!